dashboards-observability/public/components/visualizations/charts/lines/line.tsx

```diff
@@ -93,21 +92,14 @@ export const Line = ({ visualizations, layout, config }: any) => {
     xaxis = dataConfig[GROUPBY];
   }
 
-  if (isEmpty(xaxis) || isEmpty(yaxis)) return <EmptyPlaceholder icon={visMetaData?.icontype} />;
+  if (isEmpty(xaxis) || xaxis.length > 1 || isEmpty(yaxis))
```

Comment:
Wondering what happened if xaxis.length === 0? does xaxis.length !== 1 work better in this case?